src/core/storage/preferences/preferencesStorage.ts

```diff
@@ -12,6 +12,7 @@ enum PreferencesKeys {
   APP_IDENTIFIERS_FAVOURITES = "app-identifiers-favourites",
   APP_CREDS_FAVOURITES = "app-creds-favourites",
   APP_USER_NAME = "app-user-name",
+  APP_BIOMETRY = "app-biometry",
```

Comment:
I would keep anything security related to the normal DB, and not Preferences. The agent has `basicStorage` which can store items with a `MiscRecordId` - like the op password hint.

Reply:
`APP_ALREADY_INIT` isn't really comparable as this is a hack anyway - we don't want to wipe the keystore on uninstall once we release.

I'm still not sure about preferences overall because we want 1 DB to locally backup and restore (so you lose these settings when moving device) and we can always cache things in Redux for speed if needed. The one instance I like preferences is identifier/cred favourites for writing speed if someone is to repeatedly click and unclick the heart icon.

In this case, even though this is a simple flag - it's login and security related, so not something I would like to leave in a less secure place even though its a boolean. I think this will just end up growing over time.

Reply:
To reconsider in https://cardanofoundation.atlassian.net/browse/DTIS-907 - resolving this